src/OpenLoco/Title.cpp

```diff
@@ -81,13 +161,73 @@ namespace OpenLoco::Title
         initialiseViewports();
         sub_4284C8();
         Gui::init();
-        sub_444357();
+        reset();
         Gfx::invalidateScreen();
         resetScreenAge();
 
         Audio::playTitleScreenMusic();
     }
 
+    // 0x00444387
+    void update()
+    {
+        if (!isTitleMode())
+            return;
+
+        resetScreenAge();
+
+        if (_waitCounter > 0)
+        {
+            _waitCounter -= 1;
+            return;
+        }
+
+        do
+        {
+            auto& command = *_sequenceIterator++;
+            std::visit(overloaded{
+                           [](WaitStep step) {
+                               _waitCounter = step.duration / 4;
```

Comment:
```suggestion
                               // This loop slightly deviates from the original, subtract 1 tick to make up for it.
                               _waitCounter = (step.duration / 4) - 1;
```
We could also compensate the fact here that the command consumes 1 tick by simply doing this.

Reply:
Looks like a sensible compromise to me. Any thoughts about this suggestion, @duncanspumpkin?